<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-TECD7PI" failures="1" tests="13" name="Organization_ModuleTest.CreateNewOrgTest" time="223.632" errors="1" timestamp="2025-04-22T11:21:17 IST" skipped="10">
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithIndAndType" time="22.800">
    <skipped/>
    <system-out>
      <![CDATA[createOrgWithIndAndType----started--
createOrgWithIndAndType--skipped--]]>
    </system-out>
  </testcase> <!-- createOrgWithIndAndType -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithPhnNoTest" time="11.532">
    <skipped/>
    <system-out>
      <![CDATA[createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--skipped--]]>
    </system-out>
  </testcase> <!-- createOrgWithPhnNoTest -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithPhnNoTest" time="11.194">
    <skipped/>
    <system-out>
      <![CDATA[createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--skipped--]]>
    </system-out>
  </testcase> <!-- createOrgWithPhnNoTest -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgTest" time="16.730">
    <system-out>
      <![CDATA[createOrgTest----started--
createOrgTest--succes--]]>
    </system-out>
  </testcase> <!-- createOrgTest -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithIndAndType" time="20.417">
    <skipped/>
    <system-out>
      <![CDATA[createOrgWithIndAndType----started--
createOrgWithIndAndType--skipped--]]>
    </system-out>
  </testcase> <!-- createOrgWithIndAndType -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithIndAndType" time="21.888">
    <skipped/>
    <system-out>
      <![CDATA[createOrgWithIndAndType----started--
createOrgWithIndAndType--skipped--]]>
    </system-out>
  </testcase> <!-- createOrgWithIndAndType -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithPhnNoTest" time="15.741">
    <skipped/>
    <system-out>
      <![CDATA[createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--skipped--]]>
    </system-out>
  </testcase> <!-- createOrgWithPhnNoTest -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithIndAndType" time="21.041">
    <skipped/>
    <system-out>
      <![CDATA[createOrgWithIndAndType----started--
createOrgWithIndAndType--skipped--]]>
    </system-out>
  </testcase> <!-- createOrgWithIndAndType -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithPhnNoTest" time="13.409">
    <skipped/>
    <system-out>
      <![CDATA[createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--skipped--]]>
    </system-out>
  </testcase> <!-- createOrgWithPhnNoTest -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithPhnNoTest" time="12.717">
    <skipped/>
    <system-out>
      <![CDATA[createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--skipped--]]>
    </system-out>
  </testcase> <!-- createOrgWithPhnNoTest -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithPhnNoTest" time="13.364">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1067)
at org.testng.Assert.assertEquals(Assert.java:1031)
at org.testng.Assert.assertEquals(Assert.java:1077)
at Organization_ModuleTest.CreateNewOrgTest.createOrgWithPhnNoTest(CreateNewOrgTest.java:121)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
    <system-out>
      <![CDATA[createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--failed--]]>
    </system-out>
  </testcase> <!-- createOrgWithPhnNoTest -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithIndAndType" time="20.587">
    <skipped/>
    <system-out>
      <![CDATA[createOrgWithIndAndType----started--
createOrgWithIndAndType--skipped--]]>
    </system-out>
  </testcase> <!-- createOrgWithIndAndType -->
  <testcase classname="Organization_ModuleTest.CreateNewOrgTest" name="createOrgWithIndAndType" time="22.212">
    <error message="Cannot locate option with value: Electronics3784
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: &#039;4.27.0&#039;, revision: &#039;d6e718d134&#039;
System info: os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;23.0.1&#039;
Driver info: driver.version: unknown" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Cannot locate option with value: Electronics3784
For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
Build info: version: '4.27.0', revision: 'd6e718d134'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '23.0.1'
Driver info: driver.version: unknown
at org.openqa.selenium.support.ui.Select.findOptionsByValue(Select.java:393)
at org.openqa.selenium.support.ui.Select.selectByValue(Select.java:277)
at GenericUtilities.WebDriver_Utility.HandleDropdownUsingValue(WebDriver_Utility.java:60)
at Organization_ModuleTest.CreateNewOrgTest.createOrgWithIndAndType(CreateNewOrgTest.java:173)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
    <system-out>
      <![CDATA[createOrgWithIndAndType----started--
createOrgWithIndAndType--failed--]]>
    </system-out>
  </testcase> <!-- createOrgWithIndAndType -->
  <system-out>
    <![CDATA[Report configuration
Configure the DB:Connect
BT:Parallel Exe
Launch the browser
Login to the appln
createOrgTest----started--
createOrgTest--succes--
Logout of the appln
Login to the appln
createOrgWithIndAndType----started--
createOrgWithIndAndType--skipped--
Logout of the appln
Login to the appln
createOrgWithIndAndType----started--
createOrgWithIndAndType--skipped--
Logout of the appln
Login to the appln
createOrgWithIndAndType----started--
createOrgWithIndAndType--skipped--
Logout of the appln
Login to the appln
createOrgWithIndAndType----started--
createOrgWithIndAndType--skipped--
Logout of the appln
Login to the appln
createOrgWithIndAndType----started--
createOrgWithIndAndType--skipped--
Logout of the appln
Login to the appln
createOrgWithIndAndType----started--
createOrgWithIndAndType--failed--
Logout of the appln
Login to the appln
createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--skipped--
Logout of the appln
Login to the appln
createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--skipped--
Logout of the appln
Login to the appln
createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--skipped--
Logout of the appln
Login to the appln
createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--skipped--
Logout of the appln
Login to the appln
createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--skipped--
Logout of the appln
Login to the appln
createOrgWithPhnNoTest----started--
createOrgWithPhnNoTest--failed--
Logout of the appln
Close the browser
AT:Parallel Exe
Close DB connection]]>
  </system-out>
</testsuite> <!-- Organization_ModuleTest.CreateNewOrgTest -->
